src/features/call/store.ts

```diff
@@ -0,0 +1,52 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { ZetkinAction, ZetkinCall } from './types';
+import {
+  remoteItem,
+  RemoteItem,
+  remoteList,
+  RemoteList,
+} from 'utils/storeUtils';
+
+export interface CallStoreSlice {
+  activeCampaignsList: RemoteList<ZetkinAction>;
```

Comment:
This sounds weird - the word `campaign` is usually used to reflect something of the type `ZetkinCampaign`, but here it's for a list of the type `ZetkinAction`  (which I also have comments about in the types file) . 

Is there a reason why this is called "campaignsList" when it is actually a list of events?

Reply:
You're right, I changed it for `activeEventList`